tests/pdb_test.go

```diff
@@ -0,0 +1,115 @@
+package tests_test
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"path/filepath"
+	"time"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
+	policyv1 "k8s.io/api/policy/v1"
+	"k8s.io/utils/pointer"
+
+	expect "github.com/google/goexpect"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/intstr"
+
+	v1 "kubevirt.io/api/core/v1"
+
+	"kubevirt.io/kubevirt/pkg/libvmi"
+	kvpointer "kubevirt.io/kubevirt/pkg/pointer"
+	"kubevirt.io/kubevirt/tests/console"
+	"kubevirt.io/kubevirt/tests/decorators"
+	"kubevirt.io/kubevirt/tests/exec"
+	"kubevirt.io/kubevirt/tests/framework/kubevirt"
+	"kubevirt.io/kubevirt/tests/libpod"
+	"kubevirt.io/kubevirt/tests/libvmifact"
+	"kubevirt.io/kubevirt/tests/testsuite"
+)
+
+var _ = Describe("[sig-compute]Pod Disruption Budget (PDB)", decorators.SigCompute, func() {
+	var (
+		vmi *v1.VirtualMachineInstance
+		err error
+		pdb *policyv1.PodDisruptionBudget
+	)
+
+	BeforeEach(func() {
+		vmi = libvmifact.NewCirros(libvmi.WithLogSerialConsole(true))
```

Comment:
Why do you need the serial console?

Reply:
Refactored the code to use a VM this time with `evictionStrategy: LiveMigrate`

my approach is to login into the vm and poweroff from within to reproduce the bug